.github/workflows/pushimge-next.yaml

```
@@ -39,13 +39,13 @@ jobs:
         with:
           path: registry-repo
       - name: Setup Go environment
-        uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
+        uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
         with:
           go-version: 1.21
       - name: Set up QEMU # Enables arm64 image building
-        uses: docker/setup-qemu-action@b12b38b5fe75585e04f6aa3fd11fea2f4385ab7f 
+        uses: docker/setup-qemu-action@fcfabe005416080c2016468f7082c2a191cdeea3 
       - name: Login to Quay
-        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
+        uses: docker/login-action@0d4c9c5ea7693da7b068278f7b52bda2a190a446 # v3.2.0
```

Revert back to v3.0.0 as v3.1.0+ breaks functionality:
```suggestion
        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
```

---

resolved with https://github.com/devfile/registry/pull/445/commits/745cd6699888d3f21ce14b3b171b70e024ad2763

---

Did it just overwrite your change... @michael-valdron

---

@Jdubrick Yeah I pushed another commit and edited my last comment.